CHANGES.md

```diff
@@ -1,3 +1,12 @@
+v0.4 (unreleased)
```

Comment:
Bump to 1.0, quite a few interface changes. Could also remove .travis.yml. Looks good for a release!